src/items/constant-items.md

```diff
@@ -91,6 +91,32 @@ m!(const _: () = (););
 // const _: () = ();
 ```
 
+## Use and reference to `static` items
+
+When a constant item or constant block is defined, [`static` items] can be used, borrowed or taken address of.
+By extension, you are allowed to call methods that immutably borrows the `static` items as receivers.
```

Comment:
Yeah this should definitely explain that it is not allowed to read from or write to any mutable static (`static mut`  and `static !Freeze`).

Reply:
Do we already have a link into the reference to explain the `static !Freeze` items? It makes sense to me to make a reference here.

Reply:
Oh this is largely missing. I guess I have to expand this section in `const_eval.md`. I can do that.